tailwind.config.js

```diff
@@ -46,7 +46,10 @@ module.exports = {
         '0px 4px 6px -1px rgba(0, 0, 0, 0.05), 0px 2px 4px -1px rgba(0, 0, 0, 0.05)',
       lg: '0 2px 4px 0 rgba(0,0,0,0.03), 0 10px 40px 0 rgba(0,0,0,0.05)',
       xl: '0 2px 4px 0 rgba(0,0,0,0.05), 0 0px 40px 0 rgba(0,0,0,0.1)',
+      card:
+        '0px 10px 10px -5px rgba(0, 0, 0, 0.02), 0px 20px 25px -5px rgba(0, 0, 0, 0.03)',
       inner: 'inset 0 2px 4px 0 rgba(0, 0, 0, 0.05)',
+      '2xl': '25px 25px 50px -12px rgba(0, 0, 0, 0.03)',
```

Comment:
a more blurred and wide shadow from the designs, wider and blurrier than xl, hence 2xl